java/api/src/main/java/io/ray/api/function/PyFunction.java

```diff
@@ -19,29 +19,40 @@
  * {@code
  * // bar returns input, so we have to set the returnType to int.class if bar accepts an int
  * ObjectRef<Integer> res = actor.call(
- *    new PyRemoteFunction<>("example_package.example_module", "bar", Integer.class),
+ *    PyFunction.of("example_package.example_module", "bar", Integer.class),
  *    1);
  * Integer value = res.get();
  *
  * // bar returns input, so we have to set the returnType to String.class if bar accepts a string
  * ObjectRef<String> res = actor.call(
- *    new PyRemoteFunction<>("example_package.example_module", "bar", String.class),
+ *    PyFunction.of("example_package.example_module", "bar", String.class),
  *    "Hello world!");
  * String value = res.get();
  * }
  * </pre>
  */
-public class PyRemoteFunction<R> {
+public class PyFunction<R> {
   // The full module name of this function
   public final String moduleName;
   // The name of this function
   public final String functionName;
   // Type of the return value of this function
   public final Class<R> returnType;
 
-  public PyRemoteFunction(String moduleName, String functionName, Class<R> returnType) {
+  private PyFunction(String moduleName, String functionName, Class<R> returnType) {
     this.moduleName = moduleName;
     this.functionName = functionName;
     this.returnType = returnType;
   }
+
+  public static PyFunction<Object> of(
```

Comment:
Document this function and the arguments. 
Note, this is more important than documenting the fields. Because IDE prompt will show the documentation when users write this function.

Reply:
done